solidity/dashboard/src/sagas/subscriptions.js

```diff
@@ -809,3 +857,26 @@ export function* subscribeToLiquidityRewardsEvents() {
     )
   }
 }
+
+export function* watchLiquidityRewardNotifications() {
+  const {
+    eth: { defaultAccount },
+  } = yield getWeb3Context()
+
+  // for the first iteration update the lastNotificationRewardAmount variable in redux without showing message
```

Comment:
Hmm, maybe we should display at the first iteration? I can imagine a situation when a user first goes to the other page than `liquidity rewards` after connecting to the wallet and we should let the user know that have available rewards.

Reply:
Hmm i'm not sure how to do that, because when a user connects to the wallet for the first time the `lastNotificationRewardAmount` is equal to 0. So right now we get the current `reward` value and save it to the `lastNotificationRewardAmount`, becuase other way it will just show the notification when the reward value is greater than 0 (for the first iteration only of course). To make it work properly with on the first iteration we would have to have `lastNotificationRewardAmount` value from user's last login, and i'm not really sure how we can do that 🤔 Any idea if we can resolve this somehow?

Unless you just meant to inform the user that he has any rewards (even if it's the same amount since he last logged in). In that case we can do that and when the user connects to the wallet the notifications will be displayed when the reward amount is greater than 0.